src/main/java/org/folio/ebsconet/service/NotesService.java

```diff
@@ -50,8 +50,8 @@ public void linkCustomerNote(MappingDataHolder mappingDataHolder) {
       }
       createNote(note);
     }
-   else if(note != null){
-       deleteNote(note.getId());
+    else if (note != null) {
```

Comment:
Hi. Could you use Objects.nonNull method?

Reply:
sure